### PR TITLE
feat: visualization context menus to mantine

### PR DESCRIPTION
**Pull request by IrakliJani**
### Description:

migrates viz context menus to mantine (table + charts)

CleanShot 2023-12-06 at 20 31 10@2x
CleanShot 2023-12-06 at 20 31 18@2x
CleanShot 2023-12-06 at 20 31 25@2x
CleanShot 2023-12-06 at 20 31 34@2x
CleanShot 2023-12-06 at 20 34 03@2x
CleanShot 2023-12-06 at 20 32 59@2x
CleanShot 2023-12-06 at 20 32 43@2x
CleanShot 2023-12-06 at 20 34 57@2x
CleanShot 2023-12-06 at 20 34 53@2x
CleanShot 2023-12-06 at 21 44 34@2x
CleanShot 2023-12-06 at 21 38 04@2x
CleanShot 2023-12-06 at 19 42 29@2x
CleanShot 2023-12-06 at 19 44 26@2x

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/Explorer/ResultsCard/UrlMenuItems.tsx [hunk 1]
```diff
@@ -76,15 +76,16 @@ const UrlMenuItem: FC<{
...
             icon="open-application"
-            text={urlConfig.label}
-            labelElement={
+            rightSection={
                 error && (
                     <Tooltip label={error} position="right">
                         <Box>
-                            <Icon icon="issue" />
+                            {/* Icon */}
+                            {/* <Icon icon="issue" /> */}
```
Comment (magnew) [inline packages/frontend/src/components/Explorer/ResultsCard/UrlMenuItems.tsx:87]: Leftovers?

File: packages/frontend/src/components/common/Table/BodyCell.tsx [hunk 2]
```diff
@@ -52,68 +51,70 @@ const BodyCell: FC<CommonBodyCellProps> = ({
...
-            lazy
-            minimal
-            position={Position.BOTTOM_RIGHT}
-            hasBackdrop
-            backdropProps={{ onClick: handleDeselect }}
-            onOpening={() => handleSelect()}
+        <Menu
+            withinPortal
+            opened={selected}
+            // backdropProps={{ onClick: handleDeselect }}
```
Comment (magnew) [inline packages/frontend/src/components/common/Table/BodyCell.tsx:67]: Extra

File: scripts/count-blueprint-imports.ts [hunk 3]
```diff
@@ -3,6 +3,8 @@ import * as ts from 'typescript';
 
 const INITIAL_BLUEPRINT_IMPORTS_BEFORE_REFACTOR = 993;
 
+const isVerbose = process.argv.includes('--verbose');
```
Comment (magnew) [inline scripts/count-blueprint-imports.ts:6]: You want this in this review? Fine by me, just checking.

**Review by magnew — APPROVED** [review]
Beautiful. I tested this pretty heavily and it seems great. Perf seems potentially even better than the old one. 

I found one super minor issue that doesn't break anything but looks a little odd. If you click view underlying data, then click on one of the cells in the underlying data table, the menu renders empty there:

Screenshot 2023-12-06 at 11 45 28

Not a blocker.

**Comment by IrakliJani** [comment]
@magnew I was not able to get rid of the square in underlying data. that is basically a menu without items and logic was kinda intertwined between components... so.. I added copy value functionality that was missing from underlying data table 😆  I think that counts both as a fix and feature  😆

CleanShot 2023-12-07 at 18 39 13@2x

CleanShot 2023-12-07 at 21 05 43@2x

CleanShot 2023-12-07 at 21 05 37@2x

**Pull request merged**